xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -343,6 +345,13 @@ COverlay* CRenderer::Convert(CDVDOverlaySSA* o, double pts)
   int targetWidth = m_rv.Width();
   int targetHeight = m_rv.Height();
   int useMargin;
+  int videomode = CServiceBroker::GetGUI()->GetStereoscopicsManager().GetStereoModeOfPlayingVideo();
```

Comment:
Although `RENDER_STEREO_MODE` is an enum which can be converted to int, I would use the correct type.
```suggestion
  RENDER_STEREO_MODE videomode = CServiceBroker::GetGUI()->GetStereoscopicsManager().GetStereoModeOfPlayingVideo();
```